<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QWidget" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Type of the Similarity Measure</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="MeasureType">
          <item>
           <property name="text">
            <string>Complete</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nearest Neighbors</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>k-Means Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="kMeansType">
          <item>
           <property name="text">
            <string>Normal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Kernel</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="Cout">
        <property name="text">
         <string>Print information during the clustering.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Incremental">
        <property name="text">
         <string>Perform an incremental clustering</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Random Numbers</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="InternalRandom">
        <property name="text">
         <string>Use Internal Random Number Generator</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="textLabel1_4">
          <property name="text">
           <string>Seed value of the generator</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer24_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="Seed"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Generic k-Means</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="_11">
        <item>
         <widget class="QLabel" name="textLabel6_2_22">
          <property name="toolTip">
           <string>Set the ratio of convergence to stop the iterations of the k-Means.</string>
          </property>
          <property name="whatsThis">
           <string>The GVCA uses a k-Means as an optimisation operator. The ratio of convergence between two iterations of this k-Means can be fixed.</string>
          </property>
          <property name="text">
           <string>Convergence ratio</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer98_2_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="Convergence"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_12">
        <item>
         <widget class="QLabel" name="textLabel2">
          <property name="text">
           <string>Clusters Number</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer24_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="NbClusters"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_10">
        <item>
         <widget class="QLabel" name="textLabel6_2_2">
          <property name="toolTip">
           <string>Set the maximum number of iterations of the k-Means.</string>
          </property>
          <property name="whatsThis">
           <string>The GVCA uses a k-Means as an optimisation operator. The maximum number of steps can be fixed.</string>
          </property>
          <property name="text">
           <string>k-Means Iterations</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer98_2_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="MaxKMeans"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="title">
      <string>Kernel k-Means Clustering</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="Spherical">
        <property name="text">
         <string>Perform a Spherical k-Means</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_13">
        <item>
         <widget class="QLabel" name="textLabel6_2_23">
          <property name="toolTip">
           <string>Set the ratio of convergence to stop the iterations of the k-Means.</string>
          </property>
          <property name="whatsThis">
           <string>The GVCA uses a k-Means as an optimisation operator. The ratio of convergence between two iterations of this k-Means can be fixed.</string>
          </property>
          <property name="text">
           <string>Alpha</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer98_2_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="Alpha"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
